vllm/config.py

```diff
@@ -876,7 +876,8 @@ def __init__(
             from vllm.executor import ray_utils
             backend = "mp"
             ray_found = ray_utils.ray_is_available()
-            if cuda_device_count_stateless() < self.world_size:
+            if (torch.cuda.is_available()
```

Comment:
weight loading tests start to fail after this pr , I think this line is to blame. cc @mgoin